docs/config/config.xml

```diff
@@ -26,6 +26,11 @@
     </database>
 
     <CouchDB>
+        <dbName>loris</dbName>
```

Comment:
Can you make these similar to the database variables, the install script will soon do something similar to what is done for configuring the database